sceptre/cli/__init__.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 
+# flake8: noqa
+
```

Comment:
I have disabled flake8 in this file, because my change has tipped the cli function foul of the code complexity check, whereas I feel it would be the job of the maintainers to refactor this (if they wish) to satisfy flake8's code complexity opinion. Also, there does not appear to be any way in flake8 to say to just disable that one check for either the function or the whole file.

Reply:
@alexharv074 you can do `#noqa` in line with the function def and it will skip that specific function. 

Generally speaking - if flake8 is complaining about complexity then it is probably a good idea to refactor to help the maintainers understand the PR more easily.

Reply:
i disagree with this statement `..whereas I feel it would be the job of the maintainers to refactor this..`.  I agree with @ngfgrant here, please follow his advice.

Reply:
@ngfgrant fwiw, placing `#noqa` on the line with the function def does not seem to disable the linter for flake8, e.g.

```diff
diff --git a/sceptre/cli/__init__.py b/sceptre/cli/__init__.py
index 8382063..7d35dae 100644
--- a/sceptre/cli/__init__.py
+++ b/sceptre/cli/__init__.py
@@ -54,7 +54,7 @@ from sceptre.cli.helpers import setup_logging, catch_exceptions
     help="Deep merge keys in successive var dicts.")
 @click.pass_context
 @catch_exceptions
-def cli(
+def cli(  # noqa
         ctx,
         debug,
         directory,
@@ -65,7 +65,7 @@ def cli(
         ignore_dependencies,
         j2_extensions,
         merge_keys
-):
+):  # noqa
     """
     Sceptre is a tool to manage your cloud native infrastructure deployments.
     """
```

```
make lint
...
sceptre/cli/__init__.py:34:1: C901 'cli' is too complex (15)
```